Testscripts/Linux/STOR_VSS_ISCSI_PartitionDisks.sh

```diff
@@ -103,76 +96,9 @@ fi
 # Connect to the iSCSI Target
 iScsi_Connect
 check_exit_status "iScsi connection to $TargetIP" "exit"
-
-# Count the Number of partition present in added new Disk .
-for disk in $(cat /proc/partitions | grep sd | awk '{print $4}')
-do
-        if [[ "$disk" != "sda"*  && "$disk" != "sdb"* ]]; then
-            ((count++))
-        fi
-done
-
-((count--))
-
-# Format, Partition and mount all the new disk on this system.
-for driveName in /dev/sd*[^0-9]; do
-    #
-    # Skip /dev/sda and /dev/sdb
-    #
-    if [[ $driveName == "/dev/sda"  || $driveName == "/dev/sdb" ]] ; then
-        continue
-    fi
-
-    # Delete the exisiting partition
-    for (( c=1 ; c<=count; count--)); do
-        (echo d; echo $c ; echo ; echo w) | fdisk $driveName
-    done
-
-    # Partition Drive
-    (echo n; echo p; echo 1; echo ; echo +500M; echo ; echo w) | fdisk $driveName
-    (echo n; echo p; echo 2; echo ; echo; echo ; echo w) | fdisk $driveName
-    sts=$?
-    if [ 0 -ne ${sts} ]; then
-        echo "ERROR:  Partitioning disk Failed ${sts}"
-        SetTestStateAborted
-        UpdateSummary " Partitioning disk $driveName : Failed"
-        exit 1
-    else
-        LogMsg "Partitioning disk $driveName : Success"
-        UpdateSummary " Partitioning disk $driveName : Success"
-    fi
-
-    sleep 1
-
-# Create file sytem on it .
-    echo "y" | mkfs.$FILESYS ${driveName}1  ; echo "y" | mkfs.$FILESYS ${driveName}2
-    check_exit_status "Creating·FileSystem·$filesys·on·disk·$driveName" "exit"
-
-    sleep 1
-
-# mount the disk .
-    MountName="/mnt/1"
-    if [ ! -e ${MountName} ]; then
-        mkdir $MountName
-    fi
-    MountName1="/mnt/2"
-    if [ ! -e ${MountName1} ]; then
-        mkdir $MountName1
-    fi
-    mount  ${driveName}1 $MountName ; mount  ${driveName}2 $MountName1
-    sts=$?
-        if [ 0 -ne ${sts} ]; then
-            LogErr "mounting disk Failed ${sts}"
-            SetTestStateAborted
-            UpdateSummary " Mounting disk $driveName on $MountName: Failed"
-            exit 1
-        else
-            LogMsg "mounting disk ${driveName}1 on ${MountName}"
-            LogMsg "mounting disk ${driveName}2 on ${MountName1}"
-            UpdateSummary " Mounting disk ${driveName}1 : Success"
-            UpdateSummary " Mounting disk ${driveName}2 : Success"
-        fi
-    done
-
+delete_partition
+make_partition 2
```

Comment:
Not good to use magic number or please use MACRO with meaningful name. So it would have good readability.